### DISCUSSION TITLE
How can I change the main toolbar colours?

### DISCUSSION THREAD

**Discussion by Max0uishere** [xournalpp/xournalpp#3974] [Q&A]
![image](https://user-images.githubusercontent.com/102471538/164451033-e91937f1-2ab6-4429-82fa-2eeb8257aeb4.png)

How can I change the main toolbar colours? I have to open [Select Colour] everytime.

**Comment by Technius** [comment]
See the guide: https://xournalpp.github.io/guide/config/toolbar-colors/

If you're using Xournal++ 1.1.1, the only way to change the colors is to edit the `toolbar.ini` file.

A more user-friendly color configuration window is currently being worked on (progress tracked in #1812) and is planned for the next release.

**Reply by Max0uishere** [reply]
Thanks @Technius There is an issue for me with the dev version. When I use the pen to write on a notebook, drawing's texture looks blurry, after 2-3 sec renders it normal. Therefore I reverted back to the normal version.

**Reply by bhennion** [reply]
> There is an issue for me with the dev version. When I use the pen to write on a notebook, drawing's texture looks blurry, after 2-3 sec renders it normal.

This looks like a regression that should be fixed. Could you give more info on your setup (OS, version, screen resolution)? (ideally in a new issue, but here is fine too).

**Reply by Max0uishere** [reply]
> This looks like a regression that should be fixed. Could you give more info on your setup (OS, version, screen resolution)? (ideally in a new issue, but here is fine too).

Windows 10 Education 20H2 - Windows Feature Experience Pack 120.2212.4170.0
monitor: 1920*1200 60hz
i5-10400 - iGPU
Wacom Bamboo CTH-460 Pen

And I also notice very small lag when writing with pen. It doesn't happen in other apps such as One Note and Windows Journal (this one is the smoothest). I don't know how to describe it but It feels same like playing a game with 57fps on 60hz screen.
